README.md

```diff
@@ -35,6 +35,22 @@ EventMachine.run do
 end
 ```
 
+To pass additional parameters while subscribing, add them to the identifier hash. These will be available server side in the channel's params hash.
```

Comment:
maybe, for what you want to do, maybe you want to keep the channel name in `identifier` and just add to it? See: https://github.com/NullVoxPopuli/action_cable_client#the-action-cable-protocol